crypto/fipsmodule/bcm.c

```diff
@@ -19,11 +19,20 @@
 #include <openssl/crypto.h>
 
 #include <stdlib.h>
-#if defined(BORINGSSL_FIPS)
+#if defined(BORINGSSL_FIPS) && defined(OPENSSL_ANDROID)
 #include <sys/mman.h>
 #include <unistd.h>
 #endif
 
+// On Windows place the bcm code in a specific section that uses Grouped Sections
+// to control the order. $b section will place bcm inbetween the start/end markers
+// which are in $a and $z.
+#if defined(BORINGSSL_FIPS) && defined(OPENSSL_WINDOWS)
+#pragma code_seg(".fipstx$b")
+#pragma data_seg(".fipsda$b")
+#pragma const_seg(".fipsda$b")
```

Comment:
It's a good question, is it more confusing if it's there and doesn't do anything, or that we don't add it and we miss something in the future? I'll add it with a comment for now.

Reply:
I don't think we can include uninitialized data in the integrity check because it's uninitialized so we don't know what it's content is going to be at runtime.